litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/QueryEditingWizard/QueryEditor/index.tsx

```diff
@@ -650,6 +712,7 @@ const QueryEditor: React.FC<QueryEditorProps> = ({
                   className={classes.formControl}
                   style={{ width: '8.5rem' }}
                   color="primary"
+                  disabled
```

Comment:
Are we disabling the Resolution feature?

Reply:
Yes because currently the graphs don’t support changing resolution. It’s not required for the Prometheus queries.